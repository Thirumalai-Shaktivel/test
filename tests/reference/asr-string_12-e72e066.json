{
    "basename": "asr-string_12-e72e066",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/string_12.f90",
    "infile_hash": "901a66104feb7c6915c8318cb66949bb2ec841785932a466bbc726f5",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-string_12-e72e066.stdout",
    "stdout_hash": "d7ac7bddb71717e196bf41918f0cb6af124190eb00755d3bf21a31b7",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}