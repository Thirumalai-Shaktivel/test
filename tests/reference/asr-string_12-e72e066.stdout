(TranslationUnit (SymbolTable 1 {string_12: (Program (SymbolTable 2 {digits: (Variable 2 digits Local (ArrayRef 2 hex_digits [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 10 (Integer 4 [])) (ConstantInteger 1 (Integer 4 [])))] (Character 1 22 () []) (ConstantString "0123456789" (Character 1 22 () []))) (ConstantString "0123456789" (Character 1 22 () [])) Parameter (Character 1 22 () []) Source Public Required .false.), fullhex_digits: (Variable 2 fullhex_digits Local (ConstantString "0123456789ABCDEFabcdef" (Character 1 22 () [])) (ConstantString "0123456789ABCDEFabcdef" (Character 1 22 () [])) Parameter (Character 1 22 () []) Source Public Required .false.), hex_digits: (Variable 2 hex_digits Local (ArrayRef 2 fullhex_digits [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 16 (Integer 4 [])) (ConstantInteger 1 (Integer 4 [])))] (Character 1 22 () []) (ConstantString "0123456789ABCDEF" (Character 1 22 () []))) (ConstantString "0123456789ABCDEF" (Character 1 22 () [])) Parameter (Character 1 22 () []) Source Public Required .false.), letters: (Variable 2 letters Local (ConstantString "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" (Character 1 52 () [])) (ConstantString "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz" (Character 1 52 () [])) Parameter (Character 1 52 () []) Source Public Required .false.), lowercase: (Variable 2 lowercase Local (ArrayRef 2 letters [((ConstantInteger 27 (Integer 4 [])) () (ConstantInteger 1 (Integer 4 [])))] (Character 1 52 () []) (ConstantString "abcdefghijklmnopqrstuvwxy" (Character 1 52 () []))) (ConstantString "abcdefghijklmnopqrstuvwxy" (Character 1 52 () [])) Parameter (Character 1 52 () []) Source Public Required .false.), lowerhex_digits: (Variable 2 lowerhex_digits Local (ConstantString "0123456789abcdef" (Character 1 16 () [])) (ConstantString "0123456789abcdef" (Character 1 16 () [])) Parameter (Character 1 16 () []) Source Public Required .false.), octal_digits: (Variable 2 octal_digits Local (ArrayRef 2 digits [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 8 (Integer 4 [])) (ConstantInteger 1 (Integer 4 [])))] (Character 1 22 () []) (ConstantString "01234567" (Character 1 22 () []))) (ConstantString "01234567" (Character 1 22 () [])) Parameter (Character 1 22 () []) Source Public Required .false.), uppercase: (Variable 2 uppercase Local (ArrayRef 2 letters [((ConstantInteger 1 (Integer 4 [])) (ConstantInteger 26 (Integer 4 [])) (ConstantInteger 1 (Integer 4 [])))] (Character 1 52 () []) (ConstantString "ABCDEFGHIJKLMNOPQRSTUVWXYZ" (Character 1 52 () []))) (ConstantString "ABCDEFGHIJKLMNOPQRSTUVWXYZ" (Character 1 52 () [])) Parameter (Character 1 52 () []) Source Public Required .false.)}) string_12 [] [(Print () [(Var 2 fullhex_digits)]) (Print () [(Var 2 hex_digits)]) (Print () [(Var 2 lowerhex_digits)]) (Print () [(Var 2 digits)]) (Print () [(Var 2 octal_digits)]) (Print () [(Var 2 letters)]) (Print () [(Var 2 uppercase)]) (Print () [(Var 2 lowercase)])])}) [])
